functions/sentry-discord.js

```diff
@@ -0,0 +1,60 @@
+const COLORS = {
+  debug: parseInt('fbe14f', 16),
+  info: parseInt('2788ce', 16),
+  warning: parseInt('f18500', 16),
+  error: parseInt('e03e2f', 16),
+  fatal: parseInt('d20f2a', 16),
+};
+
+exports.handler = function (event, context, callback) {
+  try {
+    const { body } = event;
+
+    const payload = {
+      username: 'Sentry',
+      avatar_url: `https://raw.githubusercontent.com/IanMitchell/sentry-discord/master/sentry-icon.png`,
```

Comment:
Can this be updated to not be a hot-link? i.e, link to an image hosted within this repo. I'm assuming if that image is moved or renamed it would break the avatar_url.

Reply:
I've updated it to point to Sentry's actual logo url from their branding page. I don't think we want to store other company's logos in our repo though